draft-gazdag-x509-shbs.md

```diff
@@ -672,15 +518,15 @@ Certificate Authorities have high demands in order to ensure the availability
 of signature generation throughout the validity period of signing key pairs.
 
 Usual backup and restore strategies when using a stateless signature scheme
-(e.g. SLH-DSA) are to duplicate private keying material and to operate redundant
-signing devices or to store and safeguard a copy of the private keying material
-such that it can be used to set up a new signing device in case of technical
-difficulties.
+(e.g. SLH-DSA) are to duplicate private keying material and to operate
+redundant signing devices or to store and safeguard a copy of the private
+keying material such that it can be used to set up a new signing device in case
+of technical difficulties.
 
-For stateful HBS such straightforward backup and restore strategies will lead
-to OTS reuse with high probability as a correct state management is not
-guaranteed. Strategies for maintaining availability and keeping a correct state
-are described in Section 7 of [NIST-SP-800-208].
+For S-HBS such straightforward backup and restore strategies will lead to OTS
+reuse with high probability as a correct state management is not guaranteed.
+Strategies for maintaining availability and keeping a correct state are
+described in Section 7 of [SP800208].
```

Comment:
Here is a good point where we can reference the new draft on state and backup management for S-HBS